Integrations/integration-IntSights.yml

```diff
@@ -206,6 +461,23 @@ script:
                 Contents: fileName
             };
             break;
+
+        case 'intsights-send-mail':
+            // convert comma-separated to array
+            args.Emails = args.Emails.split(',');
```

Comment:
It converts a string such as "a,b,c" to a list [a, b, c]. It works. What do you mean by won't do anything?